doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -44,88 +44,93 @@ Validated Versions
+         $ mkdir ~/sos-kernel && cd ~/sos-kernel
+         $ git clone https://github.com/projectacrn/acrn-kernel
+         $ cd acrn-kernel
+         $ git checkout -b v2.2 remotes/origin/release_2.2
+         $ cp kernel_config_uefi_sos .config
+         $ make olddefconfig
+         $ make all
+         $ sudo make modules_install
+         $ sudo cp arch/x86/boot/bzImage /boot/bzImage
 
-      .. code-block:: bash
+#. Update Grub for the Debian Service VM
 
-         $ mkdir ~/sos-kernel && cd ~/sos-kernel
-         $ wget https://download.clearlinux.org/releases/30930/clear/x86_64/os/Packages/linux-iot-lts2018-sos-4.19.68-84.x86_64.rpm
-         $ sudo apt install rpm2cpio
-         $ rpm2cpio linux-iot-lts2018-sos-4.19.68-84.x86_64.rpm | cpio -idmv
-         $ sudo cp -r ~/sos-kernel/usr/lib/modules/4.19.68-84.iot-lts2018-sos /lib/modules/
-         $ sudo mkdir /boot/acrn/
-         $ sudo cp ~/sos-kernel/usr/lib/kernel/org.clearlinux.iot-lts2018-sos.4.19.68-84 /boot/acrn/
-         $ sudo vi /etc/grub.d/40_custom
-         <To add below>
-         menuentry 'ACRN Debian Service VM' {
-                 recordfail
-                 load_video
-                 insmod gzio
-                 insmod part_gpt
-                 insmod ext2
-
-         linux  /boot/acrn/org.clearlinux.iot-lts2018-sos.4.19.68-84 console=tty0 console=ttyS0 root=/dev/sda2 rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1 i915.enable_guc=0 hvlog=2M@0x1FE00000 memmap=2M\$0x1FE00000
-         }
-         $ sudo vi /etc/default/grub
-         <Specify the default grub to the ACRN Debian Service VM entry>
-         GRUB_DEFAULT=5
-         $ sudo update-grub
-         $ sudo reboot
-
-      You should see the Grub menu with the new "ACRN Debian Service VM"
-      entry. Select it and proceed to booting the platform. The system will
-      start the Debian Desktop and you can now log in (as before).
+      Update the ``/etc/grub.d/40_custom`` file as shown below.
+
+   .. note::
+      Enter the command line for the kernel in ``/etc/grub.d/40_custom`` as
+      a single line and not as multiple lines. Otherwise, the kernel will
+      fail to boot.
+
+   .. code-block:: none
+
+      menuentry "ACRN Multiboot Gebian Service VM" --id debian-service-vm {
+        recordfail
+        load_video
+        insmod gzio
+        insmod part_gpt
+        insmod ext2
+
+        search --no-floppy --fs-uuid --set 9bd58889-add7-410c-bdb7-1fbc2af9b0e1
+        echo 'loading ACRN...'
+        multiboot2 /boot/acrn/acrn.bin  root=PARTUUID="e515916d-aac4-4439-aaa0-33231a9f4d83"
+        module2 /boot/bzImage Linux_bzImage
+      }
+
+   .. note::
+      Update this to use the UUID (``--set``) and PARTUUID (``root=`` parameter)
+      (or use the device node directly) of the root partition (e.g.
+      ``/dev/nvme0n1p2). Hint: use ``sudo blkid /dev/sda*``.
```

Comment:
You have problems with unclosed double-backticks here.

Can you also stick to one type of disk, either `/dev/sda` or `/dev/nvme0`?

Reply:
Sorry, not get your point.